rfcs/20240312-tanop.md

```diff
@@ -51,8 +51,8 @@ produces a `result` tensor. Depending on the element type, does the following:
 
 ```mlir
 // %operand: [
-//            [0.0, 1.57079632],       // [0, pi/2]
-//            [3.14159265, 4.71238898] // [pi, 3pi/2]
```

Comment:
Not these two lines.  These two lines were correct.

It is the `%results` that requires fixes.

Reply:
yeh.. realized that later, and updated in latest commit., thanks